CMakeLists.txt

```diff
@@ -599,8 +599,8 @@ set(SOURCES
         cache/cache_reservation_manager.cc
         cache/clock_cache.cc
         cache/compressed_secondary_cache.cc
-        cache/fast_lru_cache.cc
         cache/lru_cache.cc
+        cache/fast_lru_cache.cc
```

Comment:
Why moving this line from 602 to 603 ? 
Alphabetically, cache/fast_lru_cache.cc should be before cache/lru_cache.cc